test/blocks/global-navigation/test-utilities.js

```diff
@@ -128,7 +131,7 @@ export const createFullGlobalNavigation = async ({
   window.fetch = stub().callsFake((url) => {
     if (url.includes('profile')) { return mockRes({ payload: defaultProfile }); }
     if (url.includes('placeholders')) { return mockRes({ payload: placeholders || defaultPlaceholders }); }
-    if (url.endsWith('large-menu.plain.html')) { return mockRes({ payload: largeMenuMock }); }
+    if (url.endsWith('large-menu.plain.html')) { return mockRes({ payload: hasCrossCloudMenu ? largeMenuMock : largeMenuWithoutCrossCloudMenu }); }
```

Comment:
Should this not be two different menu's and a different url if one has a cross cloud menu and the other doesn't ?

Reply:
Correct. Made the change 👍